Formula/macos-term-size.rb

```diff
@@ -0,0 +1,18 @@
+class MacosTermSize < Formula
+  desc "Get the terminal window size on macOS"
+  homepage "https://github.com/sindresorhus/macos-term-size"
+  url "https://github.com/sindresorhus/macos-term-size/archive/v1.0.0.tar.gz"
+  sha256 "5ec39d49a461e4495f20660609276b0630ef245bf8eb80c8447c090a5fefda3c"
+  license "MIT"
+  head "https://github.com/sindresorhus/macos-term-size.git", branch: "main"
+
+  def install
+    # https://github.com/sindresorhus/macos-term-size/blob/main/build#L6
+    system ENV.cc, "-std=c99", "term-size.c", "-o", "term-size"
+    bin.install "term-size"
+  end
+
+  test do
+    assert_match(/\d+\n\d+/, shell_output(bin/"term-size").chomp)
```

Comment:
```
 ==> Testing macos-term-size
/usr/bin/sandbox-exec -f /private/tmp/homebrew20210723-95978-wu5r9g.sb ruby -W1 -- /opt/homebrew/Library/Homebrew/test.rb /opt/homebrew/Library/Taps/homebrew/homebrew-core/Formula/macos-term-size.rb --verbose
==> /opt/homebrew/Cellar/macos-term-size/1.0.0/bin/term-size
Opening `/dev/tty` failed (6): Device not configured
```

Reply:
I keep forgetting these actions runners don't have TTYs. Makes trying to run tests locally sometimes very useless...